newton/_src/utils/heightfield.py

```
@@ -0,0 +1,95 @@
+    Args:
+        filename: Path to the heightfield file.
+        nrow: Expected number of rows.
+        ncol: Expected number of columns.
+
+    Returns:
+        (nrow, ncol) float32 array of elevation values.
+    """
+    ext = os.path.splitext(filename)[1].lower()
+
+    if ext == ".png":
+        from PIL import Image  # noqa: PLC0415
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)